src/components/TableComponents/TicketsBodyComponents/TicketsBodyComponents.tsx

```diff
@@ -0,0 +1,54 @@
+import { TableBody, TableCell } from "@mui/material";
+
+import CellTextItem from "../CellTextItem";
+import { TICKETS_DATA } from "../../../constants";
+import { ReactComponent as MoreIcon } from "../../../assets/logos/more.svg";
+
+import { BodyRow, UserBox, UserPhoto } from "./styled";
+import { priorityMarkers } from "./helper";
+
+interface ITicketsBodyComponents {
+  page: number;
+  rowsPerPage: number;
+}
+
+const TicketsBodyComponents = ({
+  page,
+  rowsPerPage,
+}: ITicketsBodyComponents) => {
+  return (
+    <TableBody>
+      {TICKETS_DATA.slice(
+        page * rowsPerPage,
+        page * rowsPerPage + rowsPerPage
+      ).map((row) => (
```

Comment:
Dont proceed with data preparations in render. Always try to make your render as pure as possible.
Do it in component's body

Reply:
also, better to select better naming, `row` is better for some element, but here `rowData` will be much to read in code below

Reply:
done